### PR TITLE
added content from museums group faqs

### PR DESCRIPTION
**Pull request by srabun**
http://museums_faq.iiif.io/community/groups/museums/
- added "About" and "Purpose" sections per other community group pages
- About section includes list of museums working with IIIF, and benefits/implications for museums generated by museums group
- links to general FAQ

http://museums_faq.iiif.io/community/faq/
- added some Qs that were generated in museums group, but apply to more than just museums

Notes: this is first iteration. template examples/info about IIIF at specific museums will likely take a bit longer to finalize and get online. Additional general FAQs will be added later, need to launch community effort to flesh out.

@mikeapp @azaroth42 feedback welcome.

### CODE REVIEW

File: source/community/faq/index.md [hunk 1]
```diff
@@ -47,6 +51,18 @@ IIIF allows for:
  * [Authentication API][auth]
  * In Progress: [IIIF for Audio/Visual materials][av]
 
+##  How does IIIF relate to my current, internal interoperability layer?
```
Comment (azaroth42) [inline source/community/faq/index.md]: Suggest: How do the IIIF APIs relate to ...

File: source/community/faq/index.md [hunk 2]
```diff
@@ -47,6 +51,18 @@ IIIF allows for:
...
+
+* Some institutions implement IIIF using a shim, or translation layer, on top of their existing interoperability layer.  In other cases, IIIF is implemented without dependencies on existing services. IIIF provides for different levels of compliance, beginning with the IIIF Image API. For more information, please see [IIIF technical details][tech-details].
```
Comment (azaroth42) [inline source/community/faq/index.md]: implement the IIIF APIs using shims

File: source/community/faq/index.md [hunk 3]
```diff
@@ -47,6 +51,18 @@ IIIF allows for:
  * [Authentication API][auth]
  * In Progress: [IIIF for Audio/Visual materials][av]
 
+##  How does IIIF relate to my current, internal interoperability layer?
+
+* Some institutions implement IIIF using a shim, or translation layer, on top of their existing interoperability layer.  In other cases, IIIF is implemented without dependencies on existing services. IIIF provides for different levels of compliance, beginning with the IIIF Image API. For more information, please see [IIIF technical details][tech-details].  
+
+## Does IIIF replace or integrate with my Digital Asset Management (DAM) software?
+
+* IIIF is a method for delivery of assets, not storage. Integration with your DAM software is dependent on your DAM provider. The IIIF community is currently in the midst of asking popular DAMS providers to support IIIF.
```
Comment (azaroth42) [inline source/community/faq/index.md]: I wouldn't put a time dependent statement in a FAQ ... we'll forget to take it out later.  An encouragement to request adoption of vendors could be appropriate though?

File: source/community/faq/index.md [hunk 4]
```diff
@@ -47,6 +51,18 @@ IIIF allows for:
...
+
+## Is IIIF an aggregator?
+
+* No, IIIF does not aggregate content.  IIIF defines APIs that allow for interoperability between digital image repositories.  Because these APIs are well documented and understood, use of IIIF can enable aggregators such as DPLA and Europeana to access your images more easily.
```
Comment (azaroth42) [inline source/community/faq/index.md]: Here IIIF == community, I think?

No, the IIIF community does not have a single aggregation platform for content. ... image repositories, including aggregators.  

The rationale here is that ArtStor is an aggregator by value (the actual images) and makes IIIF APIs available.

**Comment by azaroth42** [comment]
line by line comments added to the FAQ entries.

File: source/community/faq/index.md (hunk 4 above)
Comment (mikeapp) [inline source/community/faq/index.md]: The IIIF Consortium (vs. members)

**Comment by srabun** [comment]
@azaroth42 @mikeapp thank you for the suggestions, I have made the edits..

**Comment by zimeon** [comment]
Looks good and travis happy so merged.

**Pull request merged**